2019-06-14 09:41:04,997|ERROR|main|Please input the config path.
2019-06-14 09:41:05,728|ERROR|main|init datasource error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:734) [druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1060) [druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1056) [druid-1.0.29.jar:1.0.29]
	at com.upload.database.pool.druid.DruidPool.getConnection(DruidPool.java:35) [bin/:na]
	at com.upload.database.pool.DBHelper.openConn(DBHelper.java:337) [bin/:na]
	at com.upload.database.pool.DBHelper.executeQuery(DBHelper.java:117) [bin/:na]
	at com.upload.database.BaseDao.queryList(BaseDao.java:123) [bin/:na]
	at com.upload.database.BaseDao.queryList(BaseDao.java:109) [bin/:na]
	at com.upload.data.business.ServerListBusiness.listAll(ServerListBusiness.java:30) [bin/:na]
	at com.upload.component.ServerListComponent.initialize(ServerListComponent.java:33) [bin/:na]
	at com.upload.component.ComponentManager.addComponent(ComponentManager.java:54) [bin/:na]
	at com.upload.UploadServer.main(UploadServer.java:47) [bin/:na]
2019-06-14 09:41:05,757|ERROR|main|{dataSource-1} init error
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:734) [druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1060) [druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1056) [druid-1.0.29.jar:1.0.29]
	at com.upload.database.pool.druid.DruidPool.getConnection(DruidPool.java:35) [bin/:na]
	at com.upload.database.pool.DBHelper.openConn(DBHelper.java:337) [bin/:na]
	at com.upload.database.pool.DBHelper.executeQuery(DBHelper.java:117) [bin/:na]
	at com.upload.database.BaseDao.queryList(BaseDao.java:123) [bin/:na]
	at com.upload.database.BaseDao.queryList(BaseDao.java:109) [bin/:na]
	at com.upload.data.business.ServerListBusiness.listAll(ServerListBusiness.java:30) [bin/:na]
	at com.upload.component.ServerListComponent.initialize(ServerListComponent.java:33) [bin/:na]
	at com.upload.component.ComponentManager.addComponent(ComponentManager.java:54) [bin/:na]
	at com.upload.UploadServer.main(UploadServer.java:47) [bin/:na]
2019-06-14 09:41:05,759|ERROR|main|Exception:
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:734) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1060) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1056) ~[druid-1.0.29.jar:1.0.29]
	at com.upload.database.pool.druid.DruidPool.getConnection(DruidPool.java:35) ~[bin/:na]
	at com.upload.database.pool.DBHelper.openConn(DBHelper.java:337) [bin/:na]
	at com.upload.database.pool.DBHelper.executeQuery(DBHelper.java:117) [bin/:na]
	at com.upload.database.BaseDao.queryList(BaseDao.java:123) [bin/:na]
	at com.upload.database.BaseDao.queryList(BaseDao.java:109) [bin/:na]
	at com.upload.data.business.ServerListBusiness.listAll(ServerListBusiness.java:30) [bin/:na]
	at com.upload.component.ServerListComponent.initialize(ServerListComponent.java:33) [bin/:na]
	at com.upload.component.ComponentManager.addComponent(ComponentManager.java:54) [bin/:na]
	at com.upload.UploadServer.main(UploadServer.java:47) [bin/:na]
2019-06-14 09:41:05,771|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,776|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,789|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,795|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,800|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,805|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,816|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,826|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,832|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,837|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,866|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,880|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,885|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,891|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,899|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_60]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_60]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,907|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,914|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,920|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,928|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,934|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,941|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,949|ERROR|main|
java.lang.NullPointerException: null
	at com.upload.component.ServerListComponent.initialize(ServerListComponent.java:34) ~[bin/:na]
	at com.upload.component.ComponentManager.addComponent(ComponentManager.java:54) ~[bin/:na]
	at com.upload.UploadServer.main(UploadServer.java:47) [bin/:na]
2019-06-14 09:41:05,949|ERROR|main|服务器列表模块失败
2019-06-14 09:41:05,950|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:41:05,957|ERROR|Druid-ConnectionPool-Create-780934299|create connection error, url: jdbc:mysql://192.168.1.96:13619/db_upload_butterfly?allowMultiQueries=true&characterEncoding=UTF-8, errorCode 1049, state 42000
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'db_upload_butterfly'
	at sun.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.Util.getInstance(Util.java:384) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1054) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3562) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3494) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:920) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4000) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2186) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:787) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:49) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at sun.reflect.GeneratedConstructorAccessor4.newInstance(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_60]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_60]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:357) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:285) ~[mysql-connector-java-5.1.12-bin.jar:na]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:148) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:220) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:142) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1463) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1525) ~[druid-1.0.29.jar:1.0.29]
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2100) ~[druid-1.0.29.jar:1.0.29]
2019-06-14 09:42:31,018|ERROR|main|Please input the config path.
2019-06-14 09:42:32,987|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 09:42:32,991|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 09:42:33,242|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 09:42:33,242|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 09:42:33,242|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 09:42:33,242|ERROR|main|UploadServer has started successfully, taken 2234 millis.
2019-06-14 13:57:22,716|ERROR|main|Please input the config path.
2019-06-14 13:57:24,549|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 13:57:24,553|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 13:57:24,765|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 13:57:24,765|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 13:57:24,765|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 13:57:24,766|ERROR|main|UploadServer has started successfully, taken 2057 millis.
2019-06-14 13:59:28,255|ERROR|main|Please input the config path.
2019-06-14 13:59:29,777|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 13:59:29,779|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 13:59:29,964|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 13:59:29,965|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 13:59:29,965|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 13:59:29,966|ERROR|main|UploadServer has started successfully, taken 1718 millis.
2019-06-14 14:05:29,937|ERROR|main|Please input the config path.
2019-06-14 14:05:31,807|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 14:05:31,810|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 14:05:32,014|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 14:05:32,014|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 14:05:32,014|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 14:05:32,015|ERROR|main|UploadServer has started successfully, taken 2089 millis.
2019-06-14 14:12:04,272|ERROR|main|Please input the config path.
2019-06-14 14:12:06,080|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 14:12:06,083|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 14:12:06,272|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 14:12:06,273|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 14:12:06,273|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 14:12:06,273|ERROR|main|UploadServer has started successfully, taken 2010 millis.
2019-06-14 14:12:52,603|ERROR|main|Please input the config path.
2019-06-14 14:12:54,216|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 14:12:54,219|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 14:12:54,384|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 14:12:54,384|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 14:12:54,385|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 14:12:54,385|ERROR|main|UploadServer has started successfully, taken 1793 millis.
2019-06-14 14:13:37,255|ERROR|qtp1719072416-18|当前正在执行SQL的服务器:kuangjin,ID:3
2019-06-14 14:13:38,289|ERROR|qtp1719072416-18|执行完毕...
2019-06-14 14:14:13,110|ERROR|qtp1719072416-22|当前正在执行SQL的服务器:kuangjin,ID:3
2019-06-14 14:14:14,147|ERROR|qtp1719072416-22|执行完毕...
2019-06-14 14:15:38,755|ERROR|qtp1719072416-22|当前正在执行SQL的服务器:kuangjin,ID:3
2019-06-14 14:31:31,723|ERROR|main|Please input the config path.
2019-06-14 14:31:33,240|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 14:31:33,244|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 14:31:33,411|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 14:31:33,411|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 14:31:33,411|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 14:31:33,412|ERROR|main|UploadServer has started successfully, taken 1695 millis.
2019-06-14 16:05:50,717|ERROR|main|Please input the config path.
2019-06-14 16:05:52,271|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 16:05:52,273|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 16:05:52,441|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 16:05:52,441|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 16:05:52,441|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 16:05:52,442|ERROR|main|UploadServer has started successfully, taken 1734 millis.
2019-06-14 16:10:21,359|ERROR|qtp1076984738-20|当前正在执行SQL的服务器:96,ID:6
2019-06-14 16:10:44,448|ERROR|qtp1076984738-20|当前正在执行SQL的服务器:96,ID:6,DB:db_game_tlcs_0001
2019-06-14 16:11:19,384|ERROR|qtp1076984738-20|执行sql错误..
java.io.IOException: 文件名、目录名或卷标语法不正确。
	at java.io.WinNTFileSystem.createFileExclusively(Native Method) ~[na:1.8.0_60]
	at java.io.File.createNewFile(File.java:1012) ~[na:1.8.0_60]
	at com.upload.util.FileUtil.createFile(FileUtil.java:250) ~[bin/:na]
	at com.upload.util.JDBCUtils.querySql(JDBCUtils.java:63) ~[bin/:na]
	at com.upload.web.servlet.sql.QuerySqlServlet.execute(QuerySqlServlet.java:75) ~[bin/:na]
	at com.upload.web.servlet.BaseHandlerServlet.doPost(BaseHandlerServlet.java:89) [bin/:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707) [servlet-api-3.1.jar:3.1.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [servlet-api-3.1.jar:3.1.0]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:838) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:543) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1584) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1228) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:481) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1553) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1130) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.Server.handle(Server.java:564) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:318) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:112) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.Invocable.invokePreferred(Invocable.java:122) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.strategy.ExecutingExecutionStrategy.invoke(ExecutingExecutionStrategy.java:58) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produceConsume(ExecuteProduceConsume.java:201) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produce(ExecuteProduceConsume.java:97) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:672) ~[jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:590) ~[jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at java.lang.Thread.run(Thread.java:745) ~[na:1.8.0_60]
2019-06-14 16:27:21,379|ERROR|main|Please input the config path.
2019-06-14 16:27:22,950|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 16:27:22,953|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 16:27:23,127|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 16:27:23,127|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 16:27:23,127|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 16:27:23,128|ERROR|main|UploadServer has started successfully, taken 1759 millis.
2019-06-14 16:28:51,571|ERROR|qtp1076984738-20|执行sql错误..
java.io.IOException: 文件名、目录名或卷标语法不正确。
	at java.io.WinNTFileSystem.createFileExclusively(Native Method) ~[na:1.8.0_60]
	at java.io.File.createNewFile(File.java:1012) ~[na:1.8.0_60]
	at com.upload.util.FileUtil.createFile(FileUtil.java:250) ~[bin/:na]
	at com.upload.web.servlet.sql.QuerySqlServlet.execute(QuerySqlServlet.java:61) ~[bin/:na]
	at com.upload.web.servlet.BaseHandlerServlet.doPost(BaseHandlerServlet.java:89) [bin/:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707) [servlet-api-3.1.jar:3.1.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [servlet-api-3.1.jar:3.1.0]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:838) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:543) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1584) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1228) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:481) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1553) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1130) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.Server.handle(Server.java:564) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:318) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:112) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.Invocable.invokePreferred(Invocable.java:122) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.strategy.ExecutingExecutionStrategy.invoke(ExecutingExecutionStrategy.java:58) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produceConsume(ExecuteProduceConsume.java:201) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.run(ExecuteProduceConsume.java:133) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:672) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:590) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]
2019-06-14 16:30:39,086|ERROR|qtp1076984738-21|执行sql错误..
java.io.IOException: 文件名、目录名或卷标语法不正确。
	at java.io.WinNTFileSystem.createFileExclusively(Native Method) ~[na:1.8.0_60]
	at java.io.File.createNewFile(File.java:1012) ~[na:1.8.0_60]
	at com.upload.util.FileUtil.createFile(FileUtil.java:250) ~[bin/:na]
	at com.upload.web.servlet.sql.QuerySqlServlet.execute(QuerySqlServlet.java:61) ~[bin/:na]
	at com.upload.web.servlet.BaseHandlerServlet.doPost(BaseHandlerServlet.java:89) [bin/:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707) [servlet-api-3.1.jar:3.1.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [servlet-api-3.1.jar:3.1.0]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:838) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:543) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1584) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1228) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:481) [jetty-servlet-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1553) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1130) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.Server.handle(Server.java:564) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:318) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251) [jetty-server-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:112) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124) [jetty-io-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:672) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:590) [jetty-util-9.4.0.v20161208.jar:9.4.0.v20161208]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]
2019-06-14 16:33:15,543|ERROR|main|Please input the config path.
2019-06-14 16:33:17,214|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 16:33:17,216|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 16:33:17,407|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 16:33:17,408|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 16:33:17,408|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 16:33:17,408|ERROR|main|UploadServer has started successfully, taken 1872 millis.
2019-06-14 16:34:05,581|ERROR|qtp1006751649-25|当前正在执行SQL的服务器: 96 ,ID:6 ,DB:db_game_tlcs_0001
2019-06-14 16:36:22,988|ERROR|qtp1006751649-25|区服执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614163345.txt,size:1432
2019-06-14 16:36:22,989|ERROR|qtp1006751649-25|查询执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614163345.txt
2019-06-14 16:38:13,941|ERROR|qtp1006751649-21|当前正在执行SQL的服务器: 96 ,ID:6 ,DB:db_game_tlcs_0001
2019-06-14 16:40:05,792|ERROR|qtp1006751649-21|区服执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614163809.txt,size:1432
2019-06-14 16:40:05,792|ERROR|qtp1006751649-21|查询执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614163809.txt
2019-06-14 16:47:23,952|ERROR|main|Please input the config path.
2019-06-14 16:47:25,596|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 16:47:25,599|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 16:47:25,790|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 16:47:25,790|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 16:47:25,791|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 16:47:25,791|ERROR|main|UploadServer has started successfully, taken 1846 millis.
2019-06-14 16:48:16,660|ERROR|qtp1076984738-25|当前正在执行SQL的服务器: 96 ,ID:6 ,DB:db_game_tlcs_0001
2019-06-14 16:48:32,382|ERROR|qtp1076984738-25|区服执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614164812.txt
2019-06-14 16:48:32,385|ERROR|qtp1076984738-25|查询执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614164812.txt
2019-06-14 16:52:44,300|ERROR|main|Please input the config path.
2019-06-14 16:52:45,857|ERROR|main|****************** Component:[com.upload.database.DBPoolComponent] has started successfully.******************
2019-06-14 16:52:45,860|ERROR|main|****************** Component:[com.upload.component.ServerListComponent] has started successfully.******************
2019-06-14 16:52:46,068|ERROR|main|****************** Component:[com.upload.component.WebComponent] has started successfully.******************
2019-06-14 16:52:46,069|ERROR|main|****************** Component:[com.upload.component.CdnComponent] has started successfully.******************
2019-06-14 16:52:46,069|ERROR|main|****************** All Component has started successfully.******************
2019-06-14 16:52:46,069|ERROR|main|UploadServer has started successfully, taken 1776 millis.
2019-06-14 16:53:18,137|ERROR|qtp1076984738-23|当前正在执行SQL的服务器: 96 ,ID:6 ,DB:db_game_tlcs_0001
2019-06-14 16:53:25,904|ERROR|qtp1076984738-23|区服执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614165306.csv
2019-06-14 16:53:25,905|ERROR|qtp1076984738-23|查询执行完毕...数据保存:H:\web\uploadTool\UploadServer\save\20190614165306.csv
